lib/easypost/models/shipment.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class EasyPost::Models::Shipment < EasyPost::Models::EasyPostObject
+  # Get the lowest rate of a Shipment (can exclude by having `'!'` as the first element of your optional filter lists).
+  def lowest_rate(carriers = [], services = [])
+    EasyPost::Util.get_lowest_object_rate(self, carriers, services)
+  end
+
+  # Get the lowest smartrate of a Shipment.
+  def lowest_smartrate(delivery_days, delivery_accuracy)
```

Comment:
issue: we should be adjusting the verbiage of `smartrate` to either `SmartRate` in comments or `smart_rate` in function calls

Reply:
This function will be removed from the `Shipment` model, but I have adjust elsewhere that has verbiage of `smartrate`